learning_resources/models.py

```diff
@@ -78,6 +78,7 @@ class LearningResourceOfferor(TimestampedModel):
     certifications = ArrayField(models.CharField(max_length=128), default=list)
     content_types = ArrayField(models.CharField(max_length=128), default=list)
     more_information = models.URLField(blank=True)
+    value_prop = models.TextField(blank=True)
```

Comment:
Could you add a comment about this being "value proposition"? 

It would be nice if it showed up in the OpenAPI... Maybe if there's a django helptext or description for models? I'm not sure. *But a comment in code would be fine.*